clang/test/Modules/pr130712.cppm

```diff
@@ -0,0 +1,35 @@
+// RUN: rm -rf %t
+// RUN: mkdir -p %t
+// RUN: split-file %s %t
+
+// There are two requirements here to result in the owner of a macro being null.
+// 1) There must be a configuration mismatch between a header and a file it depends on
+// 2) -fmodules-local-submodule-visibility must be enabled.
+
+// Unfortunately, libcxx is compiled with exceptions, but the sysroot it depends on is likely compiled without exceptions.
```

Comment:
Why libcxx is relevant here while any code in this test doesn't include STL header?

Reply:
You *can* reproduce this with libcxx, but this is a minimal repro.

Reply:
The comment is not relevant to the test; can you remove it?